### ISSUE TITLE
Problem with Xwt.Wpf Menu-Height looks ugly :O

### ISSUE THREAD

**Issue by sourceskyboxer** [mono/xwt#774]
Hello everyone, 

please add more methods of Xwt.Menu and Xwt.MenuItem with Height Because it looks ugly why does Menu Item hides half like this
![xwt_menu_hide](https://user-images.githubusercontent.com/3496136/35962333-4454aa10-0cb1-11e8-9671-b936f41121e6.JPG)

I have tried because Menu or MenuItem doesn't have method "Height".

Is it bug? Or Wpf reference assemblies happen to mess up to Xwt?

**Comment by slluis** [comment]
I don't think we need a Heigh property, but we may need to fix the default height of a menu item.
